lib/streamlit/ReportSession.py

```diff
@@ -353,6 +353,8 @@ def _maybe_enqueue_initialize_message(self):
 
         imsg.config.mapbox_token = config.get_option("mapbox.token")
 
+        imsg.config.wide_mode = config.get_option("browser.wideMode")
```

Comment:
If we make wideMode scriptable (see👇) we should probably move this from `config` to `session_state` and update `_enqueue_session_state_changed_message` as well.